common-crafting.lic

```diff
@@ -157,7 +159,15 @@ module DRCC
 
     waitrt?
     if belt && belt['items'].find { |item| /\b#{name}/i =~ item || /\b#{item}/i =~ name }
-      DRC.bput("tie my #{name} to my #{belt['name']}", 'you attach')
+      case DRC.bput("tie my #{name} to my #{belt['name']}", 'you attach', 'Your wounds hinder')
+      when 'Your wounds hinder'
+        craft_room = Room.current.id
+        wait_for_script_to_complete('safe-room', ['force'])
+        DRCT.walk_to(craft_room)
+        return stow_crafting_item(name, bag, bag_items, belt)
```

Comment:
❓ Why does this line have four arguments but the method takes three?

Reply:
Yeah, you're absolutely right -- there is different arg handling between get_item and stow_item.  Changed that in a commit.